static/app/views/projectInstall/createProject.tsx

```diff
@@ -214,6 +217,45 @@ class CreateProject extends Component<Props, State> {
         );
         ruleId = ruleData.id;
       }
+      if (metricAlertPresets && metricAlertPresets.length > 0) {
+        const presets = PRESET_AGGREGATES.filter(aggregate =>
+          metricAlertPresets.includes(aggregate.id)
+        );
+        const teamObj = this.props.teams.find(aTeam => aTeam.slug === team);
+        for (const preset of presets) {
```

Comment:
@Neo-Zhixing should we make these API requests in parallel with `Promise.all`?

Reply:
Fixed